SeaBattle/Deck.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Deck
+    {
+        public Point Location;
+
+        // If 0 = SANK, 1 = AFLOAT.

```

Comment:
Not the best way to describe the state, at least you could use enum to describe it.

Reply:
I will change type for bool.

Reply:
When? in this PR or what?

Reply:
yes in this PR.